### PR TITLE
feat(ingest/snowflake): hide `host_port` from snowflake docs

### PR DESCRIPTION
**Pull request by hsheth2**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/configuration/pydantic_field_deprecation.py [hunk 1]
```diff
@@ -7,16 +7,14 @@
 from datahub.utilities.global_warning_util import add_global_warning
 
 
-def pydantic_field_deprecated(
```
Comment (mayurinehate) [inline metadata-ingestion/src/datahub/configuration/pydantic_field_deprecation.py:10]: Okay, so the responsibility of letting user know about the alternatives of deprecated field is entirely on the message field.

**Review by mayurinehate — APPROVED** [review]
LGTM

File: metadata-ingestion/src/datahub/configuration/pydantic_field_deprecation.py (hunk 1 above)
Comment (hsheth2) [reply metadata-ingestion/src/datahub/configuration/pydantic_field_deprecation.py:10]: yes. if you have an alternate field, use the `pydantic_renamed_field` instead

Comment (mayurinehate) [reply metadata-ingestion/src/datahub/configuration/pydantic_field_deprecation.py:10]: makes sense.

**Review by anshbansal — APPROVED** [review]

**Pull request merged**